frontend/packages/app/public/index.html

```diff
@@ -19,23 +19,20 @@
     -->
     <title>clutch</title>
 
-    <!--
-      Google Analytics
-      https://developers.google.com/analytics/devguides/collection/analyticsjs
-    -->
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=%REACT_APP_GA_TRACKING_ID%"></script>
     <script>
-      if ("%NODE_ENV%" !== "development") {
-        window.ga =
-          window.ga ||
-          function() {
-            (ga.q = ga.q || []).push(arguments);
-          };
-        ga.l = +new Date();
-        ga("create", "%REACT_APP_GA_TRACKING_ID%", "auto");
-        ga("send", "pageview");
```

Comment:
Does `config` handle these now or are they no longer necessary? I'm not terribly familiar with initial GA setup

Reply:
Yeah config handles these in GA 4.

Reply:
Cool!